react/features/recording/components/LiveStream/native/GoogleSigninForm.js

```diff
@@ -74,18 +74,6 @@ class GoogleSigninForm extends Component<Props> {
      * @inheritdoc
      */
     componentDidMount() {
-        if (Platform.OS === 'ios') {
```

Comment:
What did you do to make it work on iOS 10?

Reply:
The previous commit, which fixes deep-linking.